File: 2727-NumberOfSeniorCitizens/2727-NumberOfSeniorCitizens.java
// Last updated: 7/1/2025, 11:04:22 PM
class Solution {
    public int countSeniors(String[] details) {
        int count = 0 ; 
        for(int i = 0 ; i < details.length ; i++){
            if(Integer.parseInt(details[i].substring(11 , 13)) > 60){
                count++; 
            }
        }
        return count; 
    }
}

File: 2727-NumberOfSeniorCitizens/2727-NumberOfSeniorCitizens.py
# Last updated: 7/1/2025, 11:04:16 PM
class Solution:
    def countSeniors(self, details: List[str]) -> int:
        count = 0 ;
        for i in details: 
            if(int(i[11:13]) > 60):
                count += 1
        return count